community/kernel/src/test/java/org/neo4j/kernel/impl/api/index/IndexSamplingCancellationTest.java

```diff
@@ -1,199 +0,0 @@
-/*
```

Comment:
It only tested that the in-memory test index `HashBasedIndex` sample method threw exception in a particular case. I think it was mis-written to begin with.